pkgs/servers/photoprism/package.json

```diff
@@ -0,0 +1,129 @@
+    "@babel/plugin-proposal-object-rest-spread": "^7.16.7",
+    "@babel/plugin-transform-runtime": "^7.16.8",
+    "@babel/polyfill": "^7.12.1",
+    "@babel/preset-env": "^7.16.10",
+    "@babel/register": "^7.16.9",
+    "@babel/runtime": "^7.16.7",
+    "@lcdp/offline-plugin": "^5.1.0",
+    "@vvo/tzdb": "^6.44.0",
+    "axios": "^0.26.0",
+    "axios-mock-adapter": "^1.19.0",
+    "babel-loader": "^8.2.3",
+    "babel-plugin-istanbul": "^6.1.1",
+    "browserslist": "^4.19.1",
+    "chai": "^4.3.4",
+    "chrome-finder": "^1.0.7",
+    "core-js": "^3.21.0",
+    "cross-env": "^7.0.3",
+    "css-loader": "^6.5.1",
+    "cssnano": "^5.0.17",
+    "easygettext": "^2.17.0",
+    "eslint": "^8.7.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-config-standard": "^17.0.0-1",
+    "eslint-formatter-pretty": "^4.1.0",
+    "eslint-plugin-html": "^6.2.0",
+    "eslint-plugin-import": "^2.25.4",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-prettier-vue": "^3.1.0",
+    "eslint-plugin-promise": "^6.0.0",
+    "eslint-plugin-vue": "^8.3.0",
+    "eslint-webpack-plugin": "^3.1.1",
+    "eventsource-polyfill": "^0.9.6",
+    "file-loader": "^6.2.0",
+    "file-saver": "^2.0.5",
+    "hls.js": "^1.1.3",
+    "i": "^0.3.7",
+    "karma": "^6.3.11",
+    "karma-chrome-launcher": "^3.1.0",
+    "karma-coverage-istanbul-reporter": "^3.0.3",
+    "karma-htmlfile-reporter": "^0.3.8",
+    "karma-mocha": "^2.0.1",
+    "karma-verbose-reporter": "^0.0.8",
+    "karma-webpack": "^5.0.0",
+    "luxon": "^2.3.0",
+    "maplibre-gl": "^2.1.1",
+    "material-design-icons-iconfont": "6.1.1",
+    "mini-css-extract-plugin": "^2.5.3",
+    "minimist": ">=1.2.5",
+    "mocha": "^9.0.2",
+    "node-storage-shim": "^2.0.1",
+    "photoswipe": "^4.1.3",
+    "postcss": "^8.4.6",
+    "postcss-import": "^14.0.2",
+    "postcss-loader": "^6.2.1",
+    "postcss-preset-env": "^7.4.1",
+    "postcss-reporter": "^7.0.5",
+    "postcss-url": "^10.1.3",
+    "prettier": "^2.5.1",
+    "pubsub-js": "^1.9.4",
+    "regenerator-runtime": "^0.13.9",
+    "resolve-url-loader": "^5.0.0",
+    "sass": "^1.48.0",
+    "sass-loader": "^12.4.0",
+    "server": "^1.0.37",
+    "sockette": "^2.0.6",
+    "style-loader": "^3.3.1",
+    "svg-url-loader": "^7.1.1",
+    "tar": "^6.1.11",
+    "url-loader": "^4.1.1",
+    "util": "^0.12.4",
+    "vue": "^2.6.14",
+    "vue-fullscreen": "^2.5.2",
+    "vue-gettext": "^2.1.12",
+    "vue-infinite-scroll": "^2.0.2",
+    "vue-loader": "^15.9.8",
+    "vue-loader-plugin": "^1.3.0",
+    "vue-luxon": "^0.10.0",
+    "vue-router": "^3.5.2",
+    "vue-style-loader": "^4.1.3",
+    "vue-template-compiler": "^2.6.14",
+    "vue2-filters": "^0.14.0",
+    "vuetify": "^1.5.24",
+    "webpack": "^5.66.0",
+    "webpack-bundle-analyzer": "^4.5.0",
+    "webpack-cli": "^4.9.1",
+    "webpack-hot-middleware": "^2.25.1",
+    "webpack-manifest-plugin": "^4.1.1",
+    "webpack-md5-hash": "^0.0.6",
+    "webpack-merge": "^5.8.0"
```

Comment:
It would be nice if upstream used `devDependencies` for e.g. webpack. Then you could reduce the output size by running `npm prune --production`.

Reply:
I think it makes sense too. 

Upstream says webpack already keeps the bundle small enough: https://github.com/photoprism/photoprism/issues/1342

Should we patch it?

Reply:
If you wanna submit an upstream PR, go for it!
If I read it correctly upstream wants to move to an approach of having multiple `package.json` files.